### PR TITLE
PII Check Workflow [skip validation] [skip r-cmd]

### PR DESCRIPTION
**Pull request by tomszosz**
~new action type~
New workflow for PII Checks

### CODE REVIEW

File: .github/workflows/presidio.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Presidio 🧪
```
Comment (dinakar29) [inline .github/workflows/presidio.yaml:2]:
```suggestion
name: PII Check 💳
```

File: .github/workflows/presidio.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  pull_request:
+    branches:
+      - main
+      - pre-release
+
+jobs:
+  presidio:
+    name: Presidio 🧪
+    runs-on: ubuntu-latest
+    if: "! contains(github.event.commits[0].message, '[skip presidio]')"
```
Comment (dinakar29) [inline .github/workflows/presidio.yaml:18]:
```suggestion
    if: "! contains(github.event.commits[0].message, '[skip pii]')"
```

**Review by dinakar29 — CHANGES_REQUESTED** [review]

**Comment by dinakar29** [comment]
@tomszosz Please pull in the latest changes from the `main` branch into this branch

**Comment by dinakar29** [comment]
Also, I'm going to update the Presidio action so that its output is more meaningful, maybe even add annotation capabilities.

**Comment by tomszosz** [comment]
> @tomszosz Please pull in the latest changes from the `main` branch into this branch

latest changes from `main` branch are included

**Review by dinakar29 — APPROVED** [review]

File: .github/workflows/gitleaks.yaml [hunk 3]
```diff
@@ -44,3 +44,17 @@ jobs:
       - name: Run gitleaks ☔
         run: gitleaks -v --no-git ${{ inputs.additional_args }} --path .
         shell: bash
+
+  pii-check:
+    name: PII Check 💳
+    runs-on: ubuntu-latest
+    if: "! contains(github.event.commits[0].message, '[skip pii-check]')"
+    steps:
+
```
Comment (dinakar29) [inline .github/workflows/gitleaks.yaml:53]:
```suggestion
```

**Pull request merged**